ui/apps/platform/src/Containers/Clusters/AddClusterPrompt.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {
+    Alert,
+    AlertVariant,
+    Button,
+    ButtonVariant,
+    Flex,
+    FlexItem,
+    Text,
+    TextContent,
+    TextVariants,
+} from '@patternfly/react-core';
+
+import SecurityFullColorPink from 'images/Security_FullColor_Pink.svg';
+
+function AddClusterPrompt() {
+    return (
+        <>
+            <Alert isInline variant={AlertVariant.success} title="You are ready to go!">
+                <p className="pf-u-font-weight-normal">
+                    You have successfully deployed a Red Hat Advanced Cluster Security platform. Now
+                    you can configure the clusters you want to secure.
+                </p>
+            </Alert>
+            <Flex
+                alignItems={{ default: 'alignItemsCenter' }}
+                justifyContent={{ default: 'justifyContentCenter' }}
+                className="pf-u-text-align-center"
+                direction={{ default: 'column' }}
+            >
+                <FlexItem className="pf-u-mb-0">
+                    <img src={SecurityFullColorPink} alt="" style={{ height: '430px' }} />
+                </FlexItem>
+                <FlexItem className="pf-u-w-66">
+                    <TextContent className="pf-u-mb-xl">
+                        <Text component={TextVariants.h1} data-testid="congratulations">
```

Comment:
1. Although people debate number of `h1` elements on a page for SEO, accessibility day of learning resources included live interview of screen reader user exploring a page for the first time, which included extensive use of heading hierarchy and comments about anything that diverged from semantic hierarchy. In that case skips in levels. It seems likely that multiple `h1` would have earned a comment.
2. is `data-testid` prop left over? Test asserts on element and content.